computation-local/src/main/java/com/powsybl/computation/local/LocalComputationManager.java

```diff
@@ -143,20 +144,42 @@ public OutputStream newCommonFile(String fileName) throws IOException {
     }
 
     private ExecutionReport execute(Path workingDir, List<CommandExecution> commandExecutionList, Map<String, String> variables, ExecutionMonitor monitor)
-            throws IOException, InterruptedException {
+            throws InterruptedException {
+        // TODO concurrent
         List<ExecutionError> errors = new ArrayList<>();
+        ExecutorService executionSubmitter = Executors.newCachedThreadPool();
```

Comment:
Shouldn't we use the common "threadPool" instead of creating a new one for each "execute" ?

Reply:
> Shouldn't we use the common "threadPool" instead of creating a new one for each "execute" ?
threadPool could be set by outside of LCM. We are not sure how much threads it could be used to start process simultaneously. Be default, it is a ForkJoinPool.commonPool(), in my test, it could only start 2 process simultaneously.

Reply:
> Shouldn't we use the common "threadPool" instead of creating a new one for each "execute" ?

threadPool could be set by outside of LCM. We are not sure how much threads it could be used to start process simultaneously. Be default, it is a ForkJoinPool.commonPool(), in my test, it could only start 2 process simultaneously.

Reply:
I agree with @sylvlecl: I'm not sure it is a good idea to create a new pool of threads here. What happens if the LocalComputationManager is "busy"?